### ISSUE TITLE
[inputs.mqtt_consumer]: Error in plugin: metric parse error: expected tag at 1:3: "95"

### ISSUE THREAD

**Issue by jghendrick** [influxdata/telegraf#6090]
Hello, I'm getting the error: metric parse error when receiving the data from the mqtt broker. I'm using Mosquito on a separate server, and Influxdb, telegraf on a Cernt0s 7vm. The connections all seem to be working. I can see the topics in my database, MQTT02, but no values. I have tried
`data_format = "value"`
`data_format = "float"` also
`data_format = "json"` I get different errors, so I think I should stay with `data_format = "influx"`
I am reading some sensors temp/humidity, switches and a water meter. I built sending to the MQTT broker, here are a few example topics I see published on the mqtt
`Hatchery01-out/2/2/1/0/0`   93.6  - a temp reading example
`Hatchery01-out/2/3/1/0/1`
`smartthings/Greyt Water Meter/temperature`
`smartthings/Greyt Water Meter/waterState`   none  - water is not running example
These are published in the mqtt-spy app I use.

The Hatchery01-out gateway has 6 sensors, smartthings meter has 4 and there's more.
SO here is my setup for telegraf
```toml
[[inputs.mqtt_consumer]]
servers = ["tcp://192.168.1.14:1883"]
qos = 0
connection_timeout = "30s"
topics = [
    "smartthings/Greyt Water Meter/#",
    "Hatchery01-out/#",
   ]
data_format = "influx"
```

I have radar about the influx inline format, I do not seem to have it correct. Here are the errors, thanks for any help
```
2019-07-08T19:02:32Z I! Starting Telegraf 1.11.1
2019-07-08T19:02:32Z I! Using config file: /etc/telegraf/telegraf.conf
2019-07-08T19:02:32Z I! Loaded inputs: mqtt_consumer
2019-07-08T19:02:32Z I! Loaded aggregators:
2019-07-08T19:02:32Z I! Loaded processors:
2019-07-08T19:02:32Z I! Loaded outputs: influxdb
2019-07-08T19:02:32Z I! Tags enabled: host=localhost.localdomain
2019-07-08T19:02:32Z I! [agent] Config: Interval:10s, Quiet:false, Hostname:"localhost.localdomain", Flush Interval:10s
2019-07-08T19:02:32Z I! [inputs.mqtt_consumer] Connected [tcp://192.168.1.14:1883]
2019-07-08T19:02:33Z E! [inputs.mqtt_consumer]: Error in plugin: metric parse error: expected tag at 1:8: "34901.7"
2019-07-08T19:02:33Z E! [inputs.mqtt_consumer]: Error in plugin: metric parse error: expected tag at 1:4: "0.0"
2019-07-08T19:02:33Z E! [inputs.mqtt_consumer]: Error in plugin: metric parse error: expected tag at 1:3: "95"
2019-07-08T19:02:33Z E! [inputs.mqtt_consumer]: Error in plugin: metric parse error: expected tag at 1:5: "none"
```

**Comment by glinton** [comment]
What is an example of the output/payload you are having telegraf ingest?

**Comment by danielnelson** [comment]
I think @glinton's has your answer covered, but if you are still having trouble please ask over at the [InfluxData Community](https://community.influxdata.com) site.

**Comment by jghendrick** [comment]
Thanks @glinton, your answer did help get me in the right direction.

**Comment by mohanmaniramesh** [comment]
I'm Not getting the data in InfelxDB what will be the problem

**Comment by JS-Choi513** [comment]
It's not working for me.
In my case, I ran telegraf by using docker image. 

At first, I set the ```data_format = "influx" ``` . then,
![image](https://github.com/influxdata/telegraf/assets/32115744/22050e6d-02fa-4ffb-9328-028ddd0ef3f4)
I faced same error log in this issue. 

So, I changed the my telegrf.conf below by fallowing above instruction:
```
data_format = "value"
data_type = "string"
```

And remove telegraf container and re launch. 

![image](https://github.com/influxdata/telegraf/assets/32115744/bfdb02e2-a0a5-45cf-bcc5-46f0c3720448)

Its stopped after connection. 

Of course, I cannot receive any data.

Any help?
